### DISCUSSION TITLE
Comparison between the Post-Peell (3GC)  and SeLFCAL (2GC) Sun images

### DISCUSSION THREAD

**Discussion by Victoria-Samboco** [ratt-ru/solarkat#13] [Show and tell]
Hi @o-smirnov and @IanHeywood I've good news. 

From the last meeting #12 I'm showing below the comparison between the 3CG Sun images that I showed in the last meeting and the 2GC Sun images. And definitely the 3GC in this case was not working as expected and also think because the 2GC data was already clean enough when I did 3GC I could over cleaned the data. 

These are images from 4 different scans (3,5,7, and 25). From the first 4 images you can see that scan 7 has a problem in the image which is not present in the same scan generated by the 2GC data (selfcal-data)

![image](https://user-images.githubusercontent.com/95472901/194614828-6803233b-fe8d-41d4-a011-34d3aee93e8d.png)
```Figure 1: scan 3,5,7 and 25 from the Post peel data```

![image](https://user-images.githubusercontent.com/95472901/194614905-e320f69e-cba3-4fa7-8802-e7c5c1ab3eb5.png)
```Figure 2: scan 3,5,7 and 25 from the Selfcal data```

More images from SELFCAL  (scan 21,25,27 and 29), as I said during the last meeting I had to flag scan 23 as well as 45 because there was a lot of interference and it affected the flags in the first calibration

![image](https://user-images.githubusercontent.com/95472901/194621505-146aee31-79fa-416e-a943-223fe06b306f.png)

When I got the the results from 3GC I was wondering also why the sun  appears very faint, being that by the theory that says that the Sun causes more interference in this band (which must mean that it should appear brighter than the images of the L band)...

Now I'm happy with the results:)

**Comment by IanHeywood** [comment]
Wow this is very intriguing... Amplitude solutions absorbing it somehow?
Selfcal flux suppression writ large? Keen to here Oleg's take on this, it's
a powerful illustration of the issue if that's the cause.
